src/aliceVision/sfm/LocalBundleAdjustmentGraph.hpp

```diff
@@ -149,7 +149,7 @@ class LocalBundleAdjustmentGraph
    * @param[in] removedViewsId Set of views index to remove
    * @return true if the number of removed node is equal to the size of \c removedViewsId
    */
-  bool removeViewsToTheGraph(const std::set<IndexT>& removedViewsId);
+  bool removeViews(const sfmData::SfMData& sfmData, const std::set<IndexT>& removedViewsId);
```

Comment:
`@brief Remove specific views from the LocalBA graph. It deletes the nodes corresponding to each specified view and all incident edges.`